scripts/seed-database/upsert-production-flows.js

```diff
@@ -57,18 +58,20 @@ const LOCAL_GRAPHQL_ADMIN_SECRET = process.env.HASURA_GRAPHQL_ADMIN_SECRET;
           theme
           notify_personalisation
         }
+        users {
+          id
+          first_name
+          last_name
+          email
+          is_admin
+        }
```

Comment:
`users` can only be queried by Hasura's admin role, whereas other tables that we've synced so far can be queried by the public role.

Adding authentication to the `productionGraphqlClient` defined above feels less than ideal, because this file has a number of mutations in it that would accidently get executed by the wrong client in testing, etc.